chessboard/schema.py

```
@@ -93,13 +172,14 @@ def DictOf(schema):
 
     Shamelessly copied from Checkmate and modified.
     """
-    def check(entry):
+
+    def __call__(self, entry):
         if not isinstance(entry, dict):
```

Why are these errors swallowed and logged as warnings? Why is it okay for the schema yaml file to be broken or missing? I don't think that's a good idea.